src/components/Application/ApplicationStart.tsx

```diff
@@ -10,27 +11,62 @@ export interface ApplicationStartProps {
 }
 
 const ApplicationStart: React.FC<ApplicationStartProps> = (props) => {
-  const { template: type, sections, handleClick } = props
+  const { template, sections, handleClick } = props
+  const { name, code, startTitle, startMessage } = template
 
-  return type ? (
-    <Container text>
-      <Header as="h1" content={type ? type.description : 'Create application page'} />
-      {type && (
-        <Segment>
-          {sections && (
-            <Header as="h2" content={`Number of sections in template: ${sections.length}`} />
-          )}
-          <List>
-            {sections &&
-              sections.map((section) => (
-                <List.Item key={`list-item-${section.code}`} content={section.title} />
-              ))}
-          </List>
-          <Button content={type.name} onClick={handleClick} />
-        </Segment>
-      )}
-      {!type && <Label content="No Application" />}
-    </Container>
+  return template ? (
+    <Segment.Group style={{ backgroundColor: 'Gainsboro', display: 'flex' }}>
+      <Button as="href" icon="angle left" label={{ content: name, color: 'grey' }} />
```

Comment:
Not really a major thing right now, but I feel that a "label" element might not be the best way to represent this type of link. Also, note that the Designs show this name in plural (i.e. "User Registrations"), so maybe we need a better way to represent this. (Perhaps for now we could have `name + " applications"`, so it'll show as "User Registration applications")

Reply:
Sure, I agree. I'll make this improvement now.

Reply:
Done: [894c2fe](https://github.com/openmsupply/application-manager-web-app/pull/233/commits/894c2fe49fbfe370a99c2a306203b985fec502e1)